components/UI/inputHelper.tsx

```
@@ -12,25 +10,15 @@ type InputHelperProps = {
-          title={helperText}
-          placement="top"
-        >
-          <div className="absolute top-1/2 -translate-y-1/2 right-2">
-            <InfoIcon
-              width="20px"
-              color={error ? "red" : theme.palette.grey[800]}
-            />
-          </div>
+      {helperText && (
+        <StyledToolTip className="cursor-pointer" title={helperText} placement="top" >
+          <> </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)